_test/rabbitmq.sh

```diff
@@ -49,6 +49,26 @@ test() {
   echo "Waiting for for all pods to start..."
   try_until 60 15 true "oc get sts/rabbitmq -n $NAMESPACE --template '{{ if .status.readyReplicas }}{{ eq .spec.replicas .status.readyReplicas }}{{ else }}false{{ end }}'" || kill -s TERM $TOP_PID
 
+  # Tests for issue #311
+  echo "Ensure 'hostname' package is installed"
+  hostname_rpm=$(oc -n $NAMESPACE rsh rabbitmq-0 rpm -q hostname)
+  if [ $? -eq 0 ]; then
```

Comment:
for this type of testing (i.e.: checking packages are installed correctly), should we start using bats?
- https://github.com/sstephenson/bats

Reply:
I'm definitely open to using bats, I've used it on other projects and have had a positive experience. Can you open a separate issue for this @garethahealy?